src/components/FeedbackDialog.vue

```diff
@@ -5,11 +5,11 @@ import FeedbackForm from '@/components/FeedbackForm.vue'
 <template>
   <div
     role="dialog"
-    aria-labelledby="dialogHeader"
+    aria-labelledby="efw-dialog-header"
```

Comment:
I don't think this div/dialog needs to be labelled. Both `aria-labelledby` and the `id="efw-dialog-header"` could be removed.